docs/creating-a-package.md

```diff
@@ -1,6 +1,6 @@
 # Creating Packages
 
-Packages are at the core of Atom. Nearly everything outside of the main editor
+Packages are at the core of Atom. You can write a package in Javascript or CoffeeScript. Nearly everything outside of the main editor
```

Comment:
Maybe `Packages can be written in either JavaScript or CoffeeScript.`?
Also, this line now looks a bit longer than the 80-line limit that Atom likes :).